### PR TITLE
Plot command fixed by removing heat capacity

### PR DESCRIPTION
**Pull request by IIHermanSinghII**

### CODE REVIEW

File: example_simulation/simulation_config_materialsproject.conf [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # formulas to be queried from Materialsproject. mp_Fe-S will cause all
 # materials with iron and sulfur to be downloaded and a worker json
 # configuration will be generated for each material.
-material = [gold_fcc,mp_Ar,mp_Ne]
+material = [gold_fcc,mp_C-O,mp_Fe-S]
```
Comment (andkem) [inline example_simulation/simulation_config_materialsproject.conf:8]: Should this change really included in this PR?

File: kim.log [hunk 2]
```diff
@@ -0,0 +1,7 @@
+2022-12-19:09:36:36CET * 0 * information * 0x558666aed130 * KIM_LogImplementation.cpp:110 * Log object created.  Default verbosity level is 'information'.
```
Comment (andkem) [inline kim.log:1]: Remove this file.

**Review by andkem — CHANGES_REQUESTED** [review]

**Comment by andkem** [comment]
Please also squash your commits.

File: example_simulation/simulation_config_materialsproject.conf (hunk 1 above)
Comment (IIHermanSinghII) [reply example_simulation/simulation_config_materialsproject.conf:8]: done

File: kim.log (hunk 2 above)
Comment (IIHermanSinghII) [reply kim.log:1]: done

**Review by andkem — APPROVED** [review]

**Pull request merged**